compose_test.go

```diff
@@ -84,3 +84,57 @@ func TestComposeProject_creation(t *testing.T) {
 		require.IsType(t, cmp, project)
 	})
 }
+
+func BenchmarkComposeProjectImpl(b *testing.B) {
+	ints := make([]int, 1000)
+	for i := 0; i < 1000; i++ {
+		ints[i] = i
+	}
+	batches := make(integers, 0, 1000)
+	batches = append(batches, ints...)
+	data := ep.NewDataset(batches)
```

Comment:
we don't really care about actual values. what do you think about replacing this entire block with:
```suggestion
	data := ep.NewDataset(integer.Data(1000))
```